maintainers/maintainer-list.nix

```diff
@@ -9250,6 +9250,12 @@
     githubId = 23431373;
     name = "Christoph Neidahl";
   };
+  opeik = {
+    email = "sandro@stikic.com";
+    github = "opeik";
+    githubId = 11566773;
+    name = "Sandro Stikić";
+  };
```

Comment:
Could you split the changes to this file out into a separate commit?

Reply:
Sure thing!